editor/src/utils/canvas-react-utils.ts

```diff
@@ -5,10 +5,14 @@ import { omitWithPredicate } from '../core/shared/object-utils'
 import { MapLike } from 'typescript'
 import { firstLetterIsLowerCase } from '../core/shared/string-utils'
 import { isIntrinsicHTMLElementString } from '../core/shared/element-template'
-import { UtopiaKeys } from '../core/model/utopia-constants'
+import { UtopiaKeys, UTOPIA_UID_KEY, UTOPIA_UID_PARENTS_KEY } from '../core/model/utopia-constants'
 
 const realCreateElement = React.createElement
 
+const fragmentSymbol = Symbol.for('react.fragment')
```

Comment:
can't we import these symbols from React somehow? I'm a bit worried about silent errors if they change a symbol

Reply:
These are just moved from where they were originally, I've not added them.